_posts/blog/2016-12-19-running-on-fedora-25.md

```diff
@@ -145,13 +147,13 @@ sudo systemctl enable accumulo-{master,tserver,gc,tracer,monitor}.service
 
 ### Amazon EC2 setup
 
-For a multi-node setup, we used a Fedora 25 Cloud AMI on Amazon EC2, HVM
-virtualization type, with gp2 disk type with the root volume on EBS. Launched
-using 64GB for the root volume and did not attach additional storage. The
-latest AMI, as of this writing, seems to be `ami-e5757bf2` (`us-east-1`
-availability zone). For this example, launch three instances (in the same
-security group). The default user is `fedora` for the Fedora AMIs. Don't forget
-to make sure they are fully up-to-date (`sudo dnf --refresh upgrade`).
+For a multi-node setup, the authors used a Fedora 25 Cloud AMI on Amazon EC2,
```

Comment:
My personal preference for a paragraph like this (one thats packed with facts instead of concepts) would be to convert it to a few sentences followed by a bullet-ed list.